engine/glfw/js/library_glfw.js

```diff
@@ -522,6 +523,51 @@ var LibraryGLFW = {
       GLFW.refreshJoysticks(true);
     },
 
+    onPointerLockEventChange: function(event) {
+      var pointerLockElement = document["pointerLockElement"] ||
+                            document["mozPointerLockElement"] ||
+                            document["webkitIsPointerLockElement"] ||
+                            document["msIsPointerLockElement"];
+      GLFW.isPointerLocked = !!pointerLockElement;
```

Comment:
@Jhonnyg I think that you don't need to check all those prefixes here and below, i.e. you can make the code much cleaner.

This commit https://github.com/defold/defold/commit/8e00b5eaaf6a1c75568399c0c5c8829ebefcaa48 added "await"/"async" tokens to the source code of the loader. After that, the minimum required versions of web browsers for the Defold web target are ([caniuse](https://caniuse.com/?search=await)):
Chrome 55, Edge 14, Safari 10.1, Firefox 52.

Pointer Lock API ([caniuse](https://caniuse.com/?search=pointerlock)) is available without prefixes from the browser versions:
Chrome 37, Edge 13, Safari 10.1, Firefox 50.

Reply:
Fixed, thanks!